### PR TITLE
Added colors to highlight outputs

### PR DESCRIPTION
**Pull request by SpaceLenore**
Addded color coded outputs to bash scripts. This closes #9

### CODE REVIEW

File: testing/auto_lint.sh [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
 
-echo "==> STARTING ESLINT"
+reset="\e[0m"
+bold="\e[1m"
+green="\e[92m"
+yellow="\e[93m"
+red="\e[91m"
+
+echo -e "$bold$yellow==> STARTING ESLINT$reset"
 npm run lint
```
Comment (Glassinatorn) [inline testing/auto_lint.sh:10]: Insert a space after this line

File: testing/lint_interactive.sh [hunk 2]
```diff
@@ -1,19 +1,27 @@
...
-        echo 're-running test...'
+        echo -e "$bold$yellow==> re-running test...$reset"
         npm test
     else
-        echo 'aborting autofix, returning error.'
+        echo -e "$bold$red==> aborting autofix, returning error.$reset"
         exit 1
     fi
 else
-echo '==> LINT COMPLETED WITHOUT ERRORS'
```
Comment (Glassinatorn) [inline testing/lint_interactive.sh:17]: Indent

File: testing/lint_interactive.sh [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+echo -e "$bold$green==> LINT COMPLETED WITHOUT ERRORS$reset"
```
Comment (Glassinatorn) [inline testing/lint_interactive.sh]: Indent

File: testing/lint_interactive.sh [hunk 4]
```diff
@@ -1,19 +1,27 @@
...
 exit 0
```
Comment (Glassinatorn) [inline testing/lint_interactive.sh]: Indent

**Review by Glassinatorn — COMMENTED** [review]
See comments

File: testing/lint_interactive.sh (hunk 4 above)
Comment (SpaceLenore) [reply testing/lint_interactive.sh]: fixed in commit [15a155e](https://github.com/vattenbagarna/Frontend/pull/9/commits/15a155ef54e4e93e7d1a154de299a232ea8da33b)

File: testing/lint_interactive.sh (hunk 3 above)
Comment (SpaceLenore) [reply testing/lint_interactive.sh]: fixed in commit [15a155e](https://github.com/vattenbagarna/Frontend/pull/9/commits/15a155ef54e4e93e7d1a154de299a232ea8da33b)

File: testing/auto_lint.sh (hunk 1 above)
Comment (SpaceLenore) [reply testing/auto_lint.sh:10]: fixed in commit [15a155e](https://github.com/vattenbagarna/Frontend/pull/9/commits/15a155ef54e4e93e7d1a154de299a232ea8da33b)

**Review by Glassinatorn — APPROVED** [review]
looks good

**Pull request merged**